### PR TITLE
Feature/examples from prod

### PR DESCRIPTION
**Pull request by jannistsiroyannis**

### CODE REVIEW

File: importers/src/main/groovy/whelk/importer/WhelkCopier.groovy [hunk 1]
```diff
@@ -67,18 +89,17 @@ class WhelkCopier {
         def newId = dest.baseUri.resolve(doc.shortId).toString()
         newDoc.id = newId
 
-        if (dest.storage.getMainId(newId)) {
-            dest.storeAtomicUpdate(newDoc.getShortId(), true, "xl", "WhelkCopier") {
-                it.data = newDoc.data
+        def collection = LegacyIntegrationTools.determineLegacyCollection(newDoc, dest.jsonld)
+        if (collection && collection != "definitions") {
```
Comment (lrosenstrom) [inline importers/src/main/groovy/whelk/importer/WhelkCopier.groovy]: Will LegacyIntegrationTools.determineLegacyCollection() ever return collection = "definitions"? When I was debugging this previously, it returned null for definitions-records.

Comment (jannistsiroyannis) [reply importers/src/main/groovy/whelk/importer/WhelkCopier.groovy]: You're probably right, but this doesn't hurt and (I think that) it more clearly communicates the intent to the reader this way.

File: importers/src/main/groovy/whelk/importer/WhelkCopier.groovy [hunk 2]
```diff
@@ -24,20 +24,42 @@ class WhelkCopier {
...
+            else
+                doc = source.getDocument(id)
+            if (doc == null) {
+                System.err.println("Could not load document with ID: $id");
+                continue
+            }
+            doc.baseUri = source.baseUri
+
             // this links to:
             for (relDoc in selectBySqlWhere("""id in (select dependsonid from lddb__dependencies where id = '${id}')""")) {
```
Comment (olovy) [inline importers/src/main/groovy/whelk/importer/WhelkCopier.groovy]: Should we get more than one level of dependencies? (PostgreSQLComponent.followDependencies)

Comment (jannistsiroyannis) [reply importers/src/main/groovy/whelk/importer/WhelkCopier.groovy]: Perhaps, but that would then be a separate issue to this :)

**Pull request merged**